src/pages/index.astro

```diff
@@ -6,6 +6,7 @@ import Organisers from "../sections/Organisers.astro";
 import Connect from "../sections/Connect.astro";
 import Events from "../sections/Events.astro";
 import ButtonLink from "../components/ButtonLink.astro";
+import Countdown from "../components/Countdown.astro";
```

Comment:
Ideally I would love to drop this counter not on a homepage, but on individual event template `src/pages/events/[eventId]/index.astro`.

Reply:
Done!